docs/tutorials/form-service/custom-error-messages.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Custom error messages
+layout: page
+parent: Form Service
+grand_parent: Tutorials
+nav_exclude: false
+---
+
+### Custom Error Messages
+
+To modify the standard default messages that Ajv provides for the error messages such pattern, minLength, maxLength etc to a more user friendly and descriptive error message we will need to modify the data schema in the form definition.
+
+There is a couple approaches you can take to customizing the error messages and it involves modifying the data schema
+
+The first approach is to add in a `errorMessage` keyword in the JSON data schema and map each keyword that input field is validating against inside the `errorMessage` as show below.
+
+```
+{
+  "type": "object",
+  "properties": {
+    "firstName": {
+      "type": "string",
+      "minLength": 2,
+      "maxLength": 15,
+      "pattern": "^[a-zA-Z]*$",
+      "errorMessage": {
+        "pattern": "First name must only contain letters.  ",
+        "minLength": "Minimum length for first name is 2 characters.",
+        "maxLength": "Max length for first name is 15 characters."
+      }
+    }
+
+  }
+```
+
+If you want to mix and match to either use your own custom error messages or use the default error message provided by Ajv you can exclude the keyword to use the default error message as show below.
+
+In below example only the `pattern` keyword will have a custom error message and the `minLength` and `maxLength` will have the standard Ajv error message.
```

Comment:
In the example below, only the `pattern` constraint will have a custom error message, while `minLength` and `maxLength` will have the standard Ajv error message.